apps/platform/components/pulls/CreatePullRequestModal.tsx

```diff
@@ -77,6 +81,19 @@ const CreatePullRequestModal = ({
     pullRequestMutation.mutate({ pullRequest: { title, projectSlug } });
   };
 
+  useEffect(() => {
+    if (branchQuery.data) {
```

Comment:
Why two `useEffect` can't it be done with one ?

Reply:
fixed this @chetannn